### DISCUSSION TITLE
ldap权限组怎么与archery设置的权限组自动绑定

### DISCUSSION THREAD

**Discussion by small1445** [hhyo/Archery#1621] [Q&A]
在ldap上设置了不同的分组，在archery创建了同样的权限组名。如何做到权限组自动绑定。
就像jenkins里的项目矩阵授权策略。

**Comment by hhyo** [comment]
参考这个参数AUTH_LDAP_MIRROR_GROUPS

https://django-auth-ldap.readthedocs.io/en/latest/reference.html#std:setting-AUTH_LDAP_MIRROR_GROUPS
